sql/src/main/java/io/crate/execution/dsl/projection/WriterProjection.java

```diff
@@ -95,7 +95,10 @@
         GZIP
     }
 
-
+    public enum InputFormat {
+        JSON,
+        CSV
+    }
```

Comment:
Not sure this is placed correctly. This is related only to CopyFrom, right?

Reply:
I don't think so either, but not sure where the best place for it would be. I've moved it to `FileUriCollectPhase`, but that also doesn't seem quite right. ce7b4c77